include/fastdds/rtps/writer/RTPSWriter.h

```diff
@@ -28,11 +28,11 @@
 #include <fastdds/rtps/Endpoint.h>
 #include <fastdds/rtps/attributes/HistoryAttributes.h>
 #include <fastdds/rtps/attributes/WriterAttributes.h>
-#include <fastdds/rtps/common/LocatorSelector.hpp>
 #include <fastdds/rtps/messages/RTPSMessageGroup.h>
-#include <fastdds/rtps/messages/RTPSMessageSenderInterface.hpp>
+#include <fastdds/rtps/flowcontrol/FlowController.hpp>
+#include "DeliveryRetCode.hpp"
+#include "LocatorSelectorSender.hpp"
```

Comment:
`FlowController` and `LocatorSelectorSender` are used by pointer or reference, so we can avoid including the headers and use forward declarations instead.

Getting rid of `DeliveryRetCode` would require more work (see my comment below)

Reply:
For now I'll be able to move `FlowController`.